crypto/src/main/kotlin/web5/sdk/crypto/AwsKeyManager.kt

```diff
@@ -165,11 +168,11 @@ public class AwsKeyManager @JvmOverloads constructor(
   /**
    * Return the alias of [publicKey], as was originally returned by [generatePrivateKey].
    *
-   * @param publicKey A public key in JWK (JSON Web Key) format
+   * @param publicKey A public key in Jwk (JSON Web Key) format
    * @return The alias belonging to [publicKey]
    */
-  override fun getDeterministicAlias(publicKey: JWK): String {
-    val jwkThumbprint = publicKey.computeThumbprint()
+  override fun getDeterministicAlias(publicKey: Jwk): String {
+    val jwkThumbprint = publicKey.kid ?: publicKey.computeThumbprint()
```

Comment:
i think this should just be `publicKey.computeThumbprint()` bc `kid` may not be the thumbprint which makes the alias non-deterministic